src/Microsoft.SqlTools.Hosting/Utility/CommandOptions.cs

```diff
@@ -6,7 +6,7 @@
 using System;
 using System.Globalization;
 
-namespace Microsoft.SqlTools.Hosting.Utility
+namespace Microsoft.SqlTools.Utility
```

Comment:
Any specific reason you changed this namespace?

Reply:
Only this class was present in the namespace. I changed it so it can be consistent with other utility classes.

Reply:
Hm, I think it'd make more sense to move the others to Microsoft.SqlTools.Hosting. By convention the namespace should usually match the library name as well - and we don't want to conflict with other general "Microsoft.SqlTools" namespaces.

Reply:
That will create too many file changes for this PR. I will create a follow up PR for that.

Reply:
Sounds good, thanks. You can leave this change in here for now then so at least things are consistent.